### ISSUE TITLE
Compile fails

### ISSUE THREAD

**Issue by AdiletAbylov** [ndac-todoroki/elixir-id3#4]
Hey, 
I cannot install your library due to error during `mix deps.compile`
```
mix deps.compile id3                                                                                                      rvm:ruby-2.4.1
==> id3
Compiling NIF crate :id3 (native/id3)...
Directory '/car_music_organizer/deps/id3/native/id3' could not be found.
One of the crate directories specified for compilation does not exist.

could not compile dependency :id3, "mix compile" failed. You can recompile this dependency with "mix deps.compile id3", update it with "mix deps.update id3" or clean it with "mix deps.clean id3"
```

There is no directory `deps/id3/native/id3`. 

My rust is
```rustc 1.42.0 (b8cedc004 2020-03-09)```

Elixir is `1.9.1`, Erlang is `22`.

**Comment by gen1321** [comment]
i have same problem

**Comment by gen1321** [comment]
@AdiletAbylov Fixed by using git version instead of hex

**Comment by ndac-todoroki** [comment]
@AdiletAbylov and @gen1321 ,

thanks for the issue!The `native` dir wasn't included in hex!
I've updated master and released version `1.0.1` right now at hex.pm too.
Now `id3` should be usable in your projects.
Also, since OTP22 wasn't supported in rustler `0.20.0`, I've updated that too.

thx;

ps: I've never thought someone will look at this repo, so I'm very happy!

**Comment by AdiletAbylov** [comment]
Thank you!
Now it compiles.

My old Japanese car's audio can't work with Bluetooth 😅, so I am trying to write a tool to manage mp3 files. 
So thank you for the library!
